### PR TITLE
Feature/at 8419

### PR DESCRIPTION
**Pull request by tomarnold-ccpo**
Updated all of the copyedits across the entire app

### CODE REVIEW

File: src/components/DOW/PricingDetails.vue [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <ATATRadioGroup 
       id="CurrentPaymentArrangement"
       legend="Current payment arrangement"
-      tooltipText="<strong>Reserved instances</strong> allow you to save on usage 
-        costs for a fixed term. <strong>Pay-as-you-go instances</strong> let you 
-        pay for compute capacity by the hour or second with no long-term commitments."
+      tooltipText="<b>Reserved/pre-paid/up-front instances</b> allow you to save on
```
Comment (tonysumner-ccpo) [inline src/components/DOW/PricingDetails.vue:6]: please change to `<strong>` 
https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb

File: src/steps/02-EvaluationCriteria/EvalPlan/components/CreateEvalPlanSlideOut.vue [hunk 2]
```diff
@@ -9,6 +9,24 @@
       criteria for selecting a source based on submission of a 
       quote/proposal/white paper.
     </p>
+    <h3 class="mb-5">What factors should be considered when deciding whether
```
Comment (tonysumner-ccpo) [inline src/steps/02-EvaluationCriteria/EvalPlan/components/CreateEvalPlanSlideOut.vue:12]: why was this added?  Please make sure it is tested...

File: src/steps/03-Background/CurrentEnvironment/UploadMigrationDocuments.vue [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
+            Have you completed a migration assessment, analysis, or process to identify
+            the cloud services and tools needed?
           </h1>
           <div class="copy-max-width">
             <p class="mb-8">
               If available, you can upload this supporting documentation, and we will attach it to
-              your Description of Work to be shared with the CSPs for proposal purposes. Please do
-              not upload any classified documents.
+              your Description of Work to be shared with the CSPs for proposal purposes. 
+              <b>Please do not upload any classified documents.</b>
```
Comment (tonysumner-ccpo) [inline src/steps/03-Background/CurrentEnvironment/UploadMigrationDocuments.vue:14]: Please change to `<strong>`
https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb

File: src/steps/03-Background/CurrentEnvironment/UploadSystemDocuments.vue [hunk 4]
```diff
@@ -11,7 +11,7 @@
             <p class="mb-8">
               If available, you can upload this supporting documentation, and we will attach it to
               your Description of Work to be shared with the CSPs for proposal purposes. 
-              Please do not upload any classified documents.
+              <b>Please do not upload any classified documents.</b>
```
Comment (tonysumner-ccpo) [inline src/steps/03-Background/CurrentEnvironment/UploadSystemDocuments.vue:14]: Please change to `<strong>`

File: src/steps/10-FinancialDetails/Upload7600.vue [hunk 5]
```diff
@@ -110,14 +110,14 @@ import { AttachmentServiceCallbacks } from "@/services/attachment";
 
 let generalTermsAndConditionsToolTips = "This is a unique 20-character value";
 generalTermsAndConditionsToolTips +=
-  "generated by G-Invoicing. You can find your GT&C number";
+  "generated by G-Invoicing. You can find your GT&C Number";
 generalTermsAndConditionsToolTips +=
-  " in the top section of your FS Form 7600A.";
+  " in the top section of your <b>FS Form 7600A.</b>";
```
Comment (tonysumner-ccpo) [inline src/steps/10-FinancialDetails/Upload7600.vue:115]: please add `<strong>`

File: src/steps/10-FinancialDetails/Upload7600.vue [hunk 6]
```diff
@@ -110,14 +110,14 @@ import { AttachmentServiceCallbacks } from "@/services/attachment";
...
+  "generated by G-Invoicing. You can find your GT&C Number";
 generalTermsAndConditionsToolTips +=
-  " in the top section of your FS Form 7600A.";
+  " in the top section of your <b>FS Form 7600A.</b>";
 
 let orderNumberToolTips =
   "This is a unique 20-character value generated by G-Invoicing. ";
 orderNumberToolTips +=
-  "You can find your order number in the top section of your FS Form 7600B.";
+  "You can find your Order Number in the top section of your <b>FS Form 7600B.</b>";
```
Comment (tonysumner-ccpo) [inline src/steps/10-FinancialDetails/Upload7600.vue:120]: Please add `strong`

File: src/steps/10-FinancialDetails/GInvoiceLearnMore.vue [hunk 7]
```diff
@@ -25,8 +25,8 @@
     <p>
       The financial management system aligns processes between trading partners through each step of
       the Buy/Sell lifecycle and uses a common set of terms. This will replace the various manual
-      forms used today like the Fiscal Service Form 7600A/B, Military Interdepartmental Purchase
-      Request (MIPR), etc.
+      forms used today like the Fiscal Servicen (FS) Form 7600A/B, Military Interdepartmental
```
Comment (tonysumner-ccpo) [inline src/steps/10-FinancialDetails/GInvoiceLearnMore.vue:28]: misspelled word `Fiscal Servicen (FS)`

File: src/steps/10-FinancialDetails/GInvoicing.vue [hunk 8]
```diff
@@ -40,7 +40,8 @@
               label="Order Number"
               tooltipText="This is a 20-character value (including hyphens) generated 
                 by G-Invoicing. You may also enter 22 characters (including a period 
-                and digit at the end) to reference the version number."
+                and digit at the end) to reference the version number. Your Order Number
+                is located in the top section of <b>your FS Form 7600A</b>"
```
Comment (tonysumner-ccpo) [inline src/steps/10-FinancialDetails/GInvoicing.vue:44]: please change from `<b>` to `<strong>`

**Review by tonysumner-ccpo — CHANGES_REQUESTED** [review]

File: src/steps/08-StandardsAndCompliance/BAA.vue [hunk 9]
```diff
@@ -21,7 +21,7 @@
                   which relates to the past, present, or future physical or mental 
                   health or condition of any individual. Per the Health Insurance 
                   Portability and Accountability Act of 1996 (HIPAA), a BAA is required 
-                  between the mission owner and the business associate to provide assurance 
+                  between the Mission Pwner and the business associate to provide assurance
```
[the remark this answers is not in the document]
Comment (tomarnold-ccpo) [reply src/steps/08-StandardsAndCompliance/BAA.vue:24]: Great catch! Updated to match figma.

File: src/steps/10-FinancialDetails/GInvoicing.vue (hunk 8 above)
Comment (tomarnold-ccpo) [reply src/steps/10-FinancialDetails/GInvoicing.vue:44]: Done

File: src/steps/10-FinancialDetails/GInvoiceLearnMore.vue (hunk 7 above)
Comment (tomarnold-ccpo) [reply src/steps/10-FinancialDetails/GInvoiceLearnMore.vue:28]: nice catch. Fixed

File: src/steps/10-FinancialDetails/Upload7600.vue (hunk 6 above)
Comment (tomarnold-ccpo) [reply src/steps/10-FinancialDetails/Upload7600.vue:120]: done

File: src/steps/03-Background/CurrentEnvironment/UploadSystemDocuments.vue (hunk 4 above)
Comment (tomarnold-ccpo) [reply src/steps/03-Background/CurrentEnvironment/UploadSystemDocuments.vue:14]: done

File: src/components/DOW/PricingDetails.vue (hunk 1 above)
Comment (tomarnold-ccpo) [reply src/components/DOW/PricingDetails.vue:6]: done

File: src/steps/02-EvaluationCriteria/EvalPlan/components/Callout.vue [hunk 10]
```diff
@@ -135,7 +135,7 @@ export default class Callout extends Vue {
           of how the item will contribute to the solution is provided.`
       ];
       break;
-    case "SET_LUMP_SUM":
+    case "SetLumpSum":
```
Comment (tonysumner-ccpo) [inline src/steps/02-EvaluationCriteria/EvalPlan/components/Callout.vue:138]: this needs to be set back to SET_LUMP_SUM as that is the db value

File: src/steps/02-EvaluationCriteria/EvalPlan/components/CreateEvalPlanSlideOut.vue (hunk 2 above)
Comment (tonysumner-ccpo) [inline src/steps/02-EvaluationCriteria/EvalPlan/components/CreateEvalPlanSlideOut.vue:12]: I cannot figure out where this code is being used.  It should be in the at the red mark of the screencap.

image.png

**Review by DGabriel-CCPO — APPROVED** [review]

**Review by tonysumner-ccpo — APPROVED** [review]
